src/main/java/com/mariamkatamashvili/trainerwork/controller/TrainingController.java

```diff
@@ -0,0 +1,54 @@
+package com.mariamkatamashvili.trainerwork.controller;
+
+import com.mariamkatamashvili.trainerwork.dto.WorkDTO;
+import com.mariamkatamashvili.trainerwork.dto.WorkloadDTO;
+import com.mariamkatamashvili.trainerwork.service.TrainersWorkService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/trainings")
+@RequiredArgsConstructor
+@Tag(name = "Workload API", description = "Operations related to calculating and retrieving trainer working hours")
+public class TrainingController {
```

Comment:
I am affraid that there is name mismatch as it relates to trainer workload rather then training